### ISSUE TITLE
Filters React Warning: Can't perform a React state update on an unmounted component.

### ISSUE THREAD

**Issue by benloh** [netcreateorg/netcreate-2018#181]
```
react_devtools_backend.js:2540 Warning: Can't perform a React state update on an unmounted component. This is a no-op, but it indicates a memory leak in your application. To fix, cancel all subscriptions and asynchronous tasks in the componentWillUnmount method.
```

This would appear after:
1. Start app
2. Close Filter Pane
3. Re-open Filter Pane
4. Select "contains" on one of the fields

**Comment by benloh** [comment]
The issue is that FiltersPanel is unmounted, but it is still listening to `UDATA.OnAppStateChange("FDATA", this.UpdateFilterDefs);`.

The solution is to add a `AppStateChangeOff` call to componentWillUnmount, e.g.
```
  componentWillUnmount() {
    // console.error('TBD: gracefully unsubscribe!')
    UDATA.AppStateChangeOff("FDATA", this.UpdateFilterDefs);
  }
```

**Comment by benloh** [comment]
Addressed in #179
